src/ml/neural_net/weight_init.cpp

```diff
@@ -61,7 +61,29 @@ xavier_weight_initializer::xavier_weight_initializer(
   result.output_gate_bias_fn = zero_weight_initializer();
 
   return result;
-}
+  }
```

Comment:
This curly brace is now mis-aligned. Please consider running git-clang-format (just on your changes)